src/app/(authenticated)/maintenance/page.tsx

```
@@ -0,0 +1,229 @@
+
+      {/* View Mode Tabs */}
+      <div className='flex w-full items-center gap-2 sm:gap-4 overflow-x-auto'>
+        <div className='flex w-full border-b min-w-max'>
+          <button
+            className={`px-3 sm:px-4 py-2 text-xs sm:text-sm font-medium border-b-2 transition-colors whitespace-nowrap cursor-pointer ${
+              viewMode === 'kanban'
+                ? 'border-blue-500 text-blue-500'
+                : 'border-transparent text-gray-500 hover:text-blue-600'
+            }`}
+            onClick={() => setViewMode('kanban')}
+          >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)